pkg/common/password/manager.go

```diff
@@ -3,10 +3,14 @@ package password
 import (
 	"crypto/rand"
 	"encoding/hex"
+	"fmt"
+
+	"golang.org/x/crypto/bcrypt"
 )
 
 type Manager interface {
 	GeneratePassword(length int) (string, error)
+	GenerateHtpasswd(username string, password string) (string, error)
```

Comment:
You can improve liké that if you like
```suggestion
	GenerateHtpasswd(username, password string) (string, error)
```